packages/lodestar/src/network/gossip/gossipsub.ts

```diff
@@ -160,7 +160,7 @@ export class Eth2Gossipsub extends Gossipsub {
     const topicStr = this.getGossipTopicString(topic);
     const sszType = getGossipSSZType(topic);
     const messageData = (sszType.serialize as (object: GossipTypeMap[GossipType]) => Uint8Array)(object);
-    const sentPeers = await this.publish(topicStr, messageData);
+    const sentPeers = await this.publish(topicStr, messageData).catch((_e) => 0);
```

Comment:
This looks really dangerous, we don't want to swallow errors like this unless there's a really good reason. If there is please add a nice reasoning above explaining why is it okay to silence this call

Reply:
sorry this a typo that got introduced while i was debugging